src/lib/Lib/Config.cpp

```diff
@@ -258,7 +258,7 @@ struct PublicSettingsVisitor {
     {
         for (auto& value : values)
         {
-            MRDOCS_DECL(normalizeStringPath(self, name, value, dirs, opts, usingDefault));
+            normalizeStringPath(self, name, value, dirs, opts, usingDefault);
```

Comment:
Not introduced here: this returns `unexpected` for most arguments. But this isn't reported because it's missing a `MRDOCS_TRY`. Not sure if it should, because it turns 

```
../CMakeLists.txt --generate=adoc ../include <mrdocs-root>/share/mrdocs/headers/libcxx <mrdocs-root>/share/mrdocs/headers/clang <mrdocs-root>/share/mrdocs/headers/libc-stubs
```
into
```
/__w/mrdocs/CMakeLists.txt /__w/mrdocs/mrdocs/--generate=adoc /__w/mrdocs/mrdocs/boost/libs/url/include /github/home/local/share/mrdocs/headers/libcxx /github/home/local/share/mrdocs/headers/clang /github/home/local/share/mrdocs/headers/libc-stubs
```

(in the "Generate Demos" step in CI)

Reply:
Lol... Yes. That should definitely be MRDOCS_TRY, not MRDOCS_DECL.